index.html

```diff
@@ -34,6 +34,35 @@
         <a href="#about"><p>About</p></a>
         <a href="#contact"><p>Contact</p></a>
       </nav>
+    </div>
+    <div id="overlay">s
+      <div id="popup">
+        <div class="popup-header">
+          <h4 id="popup-project-heading" class="popup-heading">This Heading</h4>
```

Comment:
- [ ] As suggested by the previous reviewer kindly make sure that all your popup projects title has the proper `font-weight` so it matches the Figma design